mysql-test/suite/innodb/t/sys_truncate.test

```diff
@@ -1,10 +1,15 @@
 --source include/have_innodb.inc
 --source include/have_sequence.inc
+# Disable the test case because valgrind may practice
+# unfair I/O scheduling and it may result in
+# out of buffer pool pages
+--source include/not_valgrind.inc
```

Comment:
A better approach would be to specify a large enough `innodb_buffer_pool_size` so that the test with Valgrind will not run out of buffer pool, or to reduce the buffer pool consumption of this test if it is possible.

Reply:
Addressed the review comments.